paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -1405,6 +1490,7 @@ async def periodically_update_progressbar():
                 while True:
                     await asyncio.sleep(60)
                     bar.update(finished_instances)
+                    print()
```

Comment:
Did this print() stops the progress bar from acting out weird??

Reply:
It seemed to help, though not entirely.